src/core/utils/UseScrollToLocation.tsx

```diff
@@ -2,19 +2,29 @@ import { useEffect, useState } from "react";
 
 // The dependency prop is given to the hook so that it can scroll based on
 // external changes (e.g. scroll when some data on the page loads).
-export const useScrollToLocation = (dependency: unknown) => {
+export const useScrollToLocation = (dependency: unknown, delay: number = 0) => {
```

Comment:
Probably my lack of React skills here, but I would have thought you'd be able to compose effects so you could do something like `useDelay(300, useScrollToLocation(locator))`.

Reply:
No longer relevant.  We now use useScrollAfterFetchWithRetry